### DISCUSSION TITLE
Backward Compatibility for .NET 6

### DISCUSSION THREAD

**Discussion by NetanelPersik** [AutoMapper/AutoMapper.Extensions.OData#224] [General]
I know .NET 6 is out of support, but is it possible to still support it for the next year using multi-targeting?

- Can something like: `<TargetFrameworks>net6.0;net7.0;net8.0;net9.0</TargetFrameworks>` work?
- We have a huge project and we chose to align all our apis using this library, we do plan to migrate to .NET 8 in the upcoming year

**Comment by BlaiseD** [comment]
Visual Studio shows security warnings for some of the unsupported frameworks/libraries - best to avoid that for all users.  We do have a branch [here](https://github.com/AutoMapper/AutoMapper.Extensions.OData/tree/DotNetVersionSix) based on the latest pre .NET 9.0 code.   We could release an  AutoMapper.AspNetCore.OData.EFCore v5.0.X from that branch - or just fork and create your own.
